persistence-modules/elasticsearch/pom.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.baeldung.jest</groupId>
+    <artifactId>jest-demo</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+    <name>jest-demo</name>
+    <description>Demo project for Jest Client for Elasticsearch</description>
+
+
```

Comment:
Also need to add one of the existing modules as a parent. A common one to use is "parent-modules".  See https://github.com/eugenp/tutorials/blob/master/persistence-modules/java-mongodb/pom.xml for example.